src/Uno.Foundation/Point.cs

```diff
@@ -20,6 +20,16 @@ public Point(double x, double y)
 			Y = y;
 		}
 
+#if __ANDROID__
+		internal static Point From(Action<int[]> getter)
```

Comment:
Please move this method to `PopupPanel`, it's too specific to make sense as a method on `Point`.

Reply:
I think this belongs in `Point`. There is multiple places in uno.ui where we pass a reference to `new int[2]` to `.GetLocation(OnScreen|InWindow)`. Having this helper method would simplify things a little:
```cs
var p = new int[2];
view.GetLocationInWindow(p);
```
->
```cs
var p = Point.From(view.GetLocationInWindow);
```